marklogic-data-hub-central/ui/e2e/cypress/fixtures/users/architect.json

```diff
@@ -0,0 +1,6 @@
+{
+  "user-name": "hc-architect",
```

Comment:
We have a testUserWithRoles() function that we can reuse to add any roles a test would need. So we have a developer, operator and a generic user. The purpose of generic user is to add roles to it as needed and test.

Do you think we can use that to test and get rid of this user?

Reply:
Thanks.  I removed the architect role and using the loginAsTestUserWithRoles() method instead.